src/commands/prompt/index.ts

```diff
@@ -114,7 +151,10 @@ async function runInternal(initialPrompt: string, options: PromptOptions) {
     messages.push({ role: 'user', content: initialPrompt });
 
     const startTimestamp = performance.now();
-    const [content, response] = await provider.getChatCompletion(config, messages);
+    const [content, response] = await provider.getChatCompletion(config, [
+      ...internalMessages,
+      ...messages,
```

Comment:
Why just not add it to regular messages? Does it break anything?

Reply:
Good point